abstractions/common/modules.py

```diff
@@ -171,25 +185,36 @@ class MarkovHead(torch.nn.Module):
             The size of the abstract state
         - num_actions : Int
             The number of actions in the environment
-    
+
     Notes:
         - This does not support continuous action spaces right now. TODO
     """
     def __init__(self, args, feature_size, num_actions):
         super(MarkovHead, self).__init__()
-        self.inverse_model = InverseModel(args, feature_size, num_actions)
+        self.discrete = hasattr(args, 'discrete') and args.discrete
```

Comment:
I'm not sure we want this to be False in the default case no? Maybe a raise ValueError is acceptable here.

Reply:
I don't like using `hasattr()` here, but at the moment the `discrete` arg only exists in the sac_parser. I think a better solution would pull that arg into the main parser, and the ideal solution would be to automatically determine whether the action space is continuous by introspecting the environment